### PR TITLE
`MkChecks` implementation

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
I started implementing a feature related to `Checks`, but I quickly realized that I lacked testing capabilities. Therefore, I decided to improve my implementation by adding correct implementations of `MkChecks` and `MkCheck`. In my pull request, I added the following changes:

1. Implemented `MkChecks` and `MkCheck`.
2. Added tests for `MkChecks` and `MkCheck`.
3. Moved the Status and Conclusion enums into the Check interface.
___
Please, let me know if that PR too big, I'll try to split it.

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you have a look, please?

File: pom.xml [hunk 1]
```diff
@@ -116,6 +116,11 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
       <artifactId>guava</artifactId>
       <version>31.1-jre</version>
     </dependency>
+    <dependency>
+      <groupId>org.cactoos</groupId>
```
Comment (yegor256) [inline pom.xml:120]: @volodya-lombrozo can't you use Guava instead? It's already in the list of dependencies and provides the same functionality of turning an `Iterable` to a `List`.

Comment (volodya-lombrozo) [reply pom.xml]: @yegor256 Done. Could you have a look, again, please?

**Comment by volodya-lombrozo** [comment]
@yegor256 Reminder

**Comment by yegor256** [comment]
@rultor merge

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you release a new version, please? `1.6.1` I guess.

**Comment by yegor256** [comment]
@rultor release, tag is `1.6.1`

**Pull request merged**